src/Orleans.Core/Streams/Predicates/DefaultStreamFilterPredicateWrapper.cs

```diff
@@ -1,9 +1,11 @@
+using Orleans.Runtime;
```

Comment:
Should we move StreamId into the root Orleans ns?

Reply:
Not sure, I kept the same namespace as `GrainId`